src/icon.rs

```diff
@@ -22,20 +24,25 @@ const ICON_SPACE: &str = "  ";
 // s#\\u[0-9a-f]*#\=eval('"'.submatch(0).'"')#
 impl Icons {
     pub fn new(theme: Theme) -> Self {
-        let display_icons = theme == Theme::Default || theme == Theme::Unicode;
-        let (icons_by_name, icons_by_extension) = if theme == Theme::Default {
-            (
-                Self::get_default_icons_by_name(),
-                Self::get_default_icons_by_extension(),
-            )
-        } else {
-            (HashMap::new(), HashMap::new())
-        };
+        let display_icons = theme == Theme::Fancy || theme == Theme::Unicode;
+        let (icons_by_name, icons_by_extension, default_file_icon, default_folder_icon) =
+            if theme == Theme::Fancy {
+                (
+                    Self::get_default_icons_by_name(),
+                    Self::get_default_icons_by_extension(),
+                    "\u{f016}",
```

Comment:
Can you add the icon representation in comment please? Like it was made before. It make the code easier to read for the peoples with unicode and/or nerd-font.

Reply:
Alright, will do.